### DISCUSSION TITLE
1.0.0

### DISCUSSION THREAD

**Discussion by AlexZeitler** [PDMLab/docker-compose#146] [Ideas]
@Steveb-p What do think about bumping version to 1.0.0 after finishing #143 #144 and #145? 

I think it's about time given the number of [dependents](https://github.com/PDMLab/docker-compose/network/dependents) and number of downloads / week?

<img width="398" alt="image" src="https://user-images.githubusercontent.com/287480/114313119-7d03d400-9af5-11eb-956f-0f19b7d32430.png">

**Comment by Steveb-p** [comment]
Sorry, I missed this discussion @AlexZeitler. It's functionality is still "new" to me.

I don't mind creating a stable version, even with incoming changes to Docker CLI. The lib wasn't changing in terms of API for a long time and, worst case scenario, you can just bump the major version as per usual by semver.

License is already present, so any liability claims should be covered by that, if something isn't still working.

**Reply by AlexZeitler** [reply]
I think it can by still useful after merging #152.
